### ISSUE TITLE
change annotation attribution

### ISSUE THREAD

**Issue by Antonialock** [pombase/canto#1182]
in session 9cf15ade89e44016 I removed a number of flocculation BPs that the community curator had made. I then decided to add back in a subset of them. When I add them back in they are attributed to me...should we add in the functionality for us to change attribution between users? 

Maybe it is not so important in the grand scheme of things...

**Comment by ValWood** [comment]
Yes we should be able to edit curator. I had a similar situation this week  where I duplicated  an annotation to capture a multigene phenotype where the  other mutant was captured as a  note
(users struggling creating genotypes!)
This was attributed to me.

Yep I think flocculation is one of the things which is controlled transcriptionally....

**Comment by kimrutherford** [comment]
Duplicate of #693?

**Comment by Antonialock** [comment]
yeah
